README.md

```diff
@@ -10,28 +10,186 @@ The features are available before an official release so that users and collabor
-- efficient and generic [replay buffers](torchrl/data/replay_buffers/replay_buffers.py);
-- [TensorDict](torchrl/data/tensordict/tensordict.py), a convenient data structure to pass data from one object to another without friction;
+TorchRL aims at a high modularity (1) and good runtime performance (2).
+
+## Features
+
+On the high-level end, TorchRL provides:
+- [`TensorDict`](torchrl/data/tensordict/tensordict.py), 
+a convenient data structure<sup>(1)</sup> to pass data from 
+one object to another without friction.
+`TensorDict` makes it easy to re-use pieces of code across environments, models and
+algorithms. For instance, here's how to code a rollout in TorchRL:
+    <details>
+      <summary>Code</summary>
+    
+    ```python
+    tensordict = env.reset()
+    policy = TensorDictModule(
+        model, 
+        in_keys=["observation_pixels", "observation_vector"],
+        out_keys=["action"],
+    )
+    out = []
+    for i in range(n_steps):
+        tensordict = policy(tensordict)
+        tensordict = env.step(tensordict)
+        out.append(tensordict)
+        tensordict = step_tensordict(tensordict)  # renames next_observation_* keys to observation_*
+    out = torch.stack(out, 0)  # TensorDict supports multiple tensor operations
+    ```
+    </details>
+
+    Check our [tutorial](tutorials/tensordict.ipynb) for more information.
 - An associated [`TensorDictModule` class](torchrl/modules/tensordict_module/common.py) which is [functorch](https://github.com/pytorch/functorch)-compatible! 
-- [interfaces for environments](torchrl/envs) from common libraries (OpenAI gym, deepmind control lab, etc.) and [wrappers for parallel execution](torchrl/envs/vec_env.py), as well as a new pytorch-first class of [tensor-specification class](torchrl/data/tensor_specs.py);
-- [environment transforms](torchrl/envs/transforms/transforms.py), which process and prepare the data coming out of the environments to be used by the agent;
-- various tools for distributed learning (e.g. [memory mapped tensors](torchrl/data/tensordict/memmap.py));
-- various [architectures](torchrl/modules/models/) and models (e.g. [actor-critic](torchrl/modules/tensordict_module/actors.py));
-- [exploration wrappers](torchrl/modules/tensordict_module/exploration.py) and [modules](torchrl/modules/models/exploration.py);
+- multiprocess [data collectors](torchrl/collectors/collectors.py)<sup>(2)</sup> that work synchronously or asynchronously:
+    <details>
+      <summary>Code</summary>
+    
+    ```python
+    collector = MultiaSyncDataCollector(
+        [make_env, make_env], 
+        policy=policy, 
+        devices=["cuda:0", "cuda:0"],
+        total_frames=10000,
+        frames_per_batch=50,
+        ...
+    )
+    for i, tensordict_data in enumerate(collector):
+        loss = loss_module(tensordict_data)
+        loss.backward()
+        optim.step()
+        optim.zero_grad()
+        collector.update_policy_weights_()
```

Comment:
Outside of changing the collector policy it seems like this is the standard pytorch training loop which was really nice. So a quick picture again would be really helpful because you're basically showing how similar RL is to standard supervised learning but where you change the data loader on the fly